pkg/agent/util/net_test.go

```diff
@@ -70,6 +70,97 @@ func TestGenerateContainerInterfaceName(t *testing.T) {
 	}
 }
 
+func TestGenerateContainerHostVethName(t *testing.T) {
+	podName0 := "pod0"
+	podNamespace0 := "ns0"
+	containerID0 := "container0"
+	eth0 := "eth0"
+	ifaceName0 := GenerateContainerHostVethName(podName0, podNamespace0, containerID0, eth0)
+	assert.True(t, len(ifaceName0) <= interfaceNameLength)
+	assert.True(t, strings.HasPrefix(ifaceName0, podName0+"-"))
```

Comment:
use `require` as you typically want all the subtests below to be skipped if one of these assertions is false

Reply:
Done